### ISSUE TITLE
Import Browser Bookmarks into Supermemory

### ISSUE THREAD

**Issue by pablomuro** [supermemoryai/supermemory#102]
As you guys, I have a lot of favorites bookmarked in my browser
If the extension or the application had the option of loading the bookmarks to the memories, it will be perfect

**Comment by Dhravya** [comment]
Hi! Thanks for this issue, i'll add this in the next version of the extension we are releasing today!

**Comment by Yh010** [comment]
I can try to solve this issue.
Been trying out chrome extension projects since few days:https://github.com/Yh010/Chrome-Extensions

**Comment by Yh010** [comment]
> I can try to solve this issue. Been trying out chrome extension projects since few days:https://github.com/Yh010/Chrome-Extensions

Implemented a basic bookmarks extension as well: https://github.com/Yh010/Chrome-Extensions/tree/main/Bookmarks-extension

**Comment by Yh010** [comment]
I have one question before starting to implement. How do we plan to show the bookmarks which are inside a folder ?
Many people group their bookmarks into folders for better organization. So how should we display such bookmarks on supermemory ?

**Comment by Dhravya** [comment]
@Yh010 maybe we can use the create space endpoint and add them to space?

GET space and POST space exist

**Comment by sagarsanjaysutar** [comment]
Any updates on this? The whole reason I signed up for this app is because you've claimed to be a bookmarking tool. Although I see you have a manual option to add URLs, we can't keep doing it for 100s of bookmarks we have already saved in our browser. 
Furthermore, if this feature is added in future, would this be in sync with the browser bookmarks? It would be tedious if the user had to always sync/manually import the bookmarks.

**Comment by sajaldebnath** [comment]
This is true, the only reason I signed up for this because it claimed to work with bookmarks. If I cannot import my existing bookmarks, then this tool is useless for me.
